app/assets/javascripts/geoblacklight/templates/index_map_info.hbs

```diff
@@ -1,8 +1,10 @@
 <div class="index-map-info">
   {{#if title}}
-    <h3>{{title}}</h3>
+    <div class="col-sm-12">
+      <h3>{{title}}</h3>
+    </div>
   {{/if}}
-  <div class="">
+  <div class="col-sm-12">
```

Comment:
Noticed that you went with stacking at small breakpoint vs medium breakpoint used by GBL's `document-metadata` dl. Was that by design?

Reply:
I did this on purpose. I wanted to make sure that it consistently kept the same width/spacing across responsive sizes. I always default to using the smaller breakpoints if it's going to be the same all the way to the smallest screen size.